tools/bin/mbedtls-archive-epic.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+""" Assist with the transition of a projectv1 to a projectv2. 
+    This is currently used for management purposes and archiving EPICS.
+
+    It will only move public issues/pr's that have been closed/merged.
+
+    Requires a fully woking gh cli https://cli.github.com/ set-up
+    and an access token with the appropriate permissions.    
+"""
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License") you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import json
+import requests
+from shlex import split
+from string import Template
+from subprocess import Popen, check_output, PIPE
+from pprint import pprint
+
+token = os.environ['GITHUB_TOKEN']
+
+
+######################### User Config ###########################
+# Instuctions
+# - Please ensure that your github cli token is present in your enviroment
+# - Set the epic name to tranfer here.
+# - Create the epic column to the arhive board, matching the src name.
+# - If needed adjust the closed_after date to be in a resonable window
+# - Fine tune max entries selection if some entries are missing.
+# - Private issues and drafts are not picked up by the script.
+
+epic = "3.6.1 patch release"       # Name of epic that needs to be moved
+                                    # (It needs to match an empty epic on v2 board)
+closed_after_date = "2024-01-01"    # Only include entries closed after that date
+
+max_entries = 500                   # How many maximum entries the backend will populate.
+debug = True                       # Set to True to print out the github cli commands.
```

Comment:
Can these please be command line items? I don't want to have to modify the script to make it do something useful. Also, because there are prefilled default, I'm afraid that I'd forget to change something and I'd end up modifying something I didn't intend.